hoot-core/src/main/cpp/hoot/core/elements/ElementData.h

```diff
@@ -27,13 +27,8 @@
 #ifndef ELEMENT_DATA_H
 #define ELEMENT_DATA_H
 
-#include "Tags.h"
-
-// Boost
-#include <boost/shared_ptr.hpp>
-#include <boost/weak_ptr.hpp>
```

Comment:
Every once in a while I came across some `#include`s that didn't need to be there so I removed them.